Automattic/Gravatar-SDK-Android
# Rename classes and methods

Closes #74

### Description

- Rename classes and methods
- Rewrite GravatarUtils  (features are now in AvatarUrl and HashExtensions)
- Introduced new "types": Hash and Email to make sure third party dev uses the right data in the right methods

### Testing Steps

- Check tests are running and are correct
- Run the demo app, check basic features (get profile, get avatar, upload avatar)

Note that I've made another version of it without Hash and Email classes: https://github.com/Automattic/Gravatar-SDK-Android/tree/maxime/74-classes-methods-rename-STRINGS
